packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -91,27 +110,16 @@ const PipelineRuns = ({
 }) => {
   let hasRunActions = false;
   const defaultHeaders = {
-    status: intl.formatMessage({
-      id: 'dashboard.tableHeader.status',
-      defaultMessage: 'Status'
-    }),
-    name: intl.formatMessage({
-      id: 'dashboard.tableHeader.name',
-      defaultMessage: 'Name'
-    }),
     pipeline: intl.formatMessage({
       id: 'dashboard.tableHeader.pipeline',
       defaultMessage: 'Pipeline'
     }),
-    namespace: 'Namespace',
-    createdTime: intl.formatMessage({
-      id: 'dashboard.tableHeader.createdTime',
-      defaultMessage: 'Created'
+    run: 'Run',
```

Comment:
Just to confirm, should this be extracted or treating as a special word that is never translated?

Reply:
special word, similar to 'PipelineRun' 👍